src/main/java/frc/robot/subsystems/swerve/factories/swerveconstants/RealSwerveConstants.java

```diff
@@ -3,16 +3,19 @@
 import com.pathplanner.lib.util.PIDConstants;
 import edu.wpi.first.math.geometry.Rotation2d;
 import frc.robot.subsystems.swerve.SwerveConstants;
+import frc.robot.subsystems.swerve.SwerveName;
 
 public class RealSwerveConstants {
 
     public static final double VELOCITY_AT_12_VOLTS_METERS_PER_SECOND = 5.054;
+    public static final SwerveName SWERVE_NAME = SwerveName.SWERVE;
```

Comment:
do the same as in sim constants

Reply:
in sim each different swerve will have the same constants, but in real swerve1 and swerve2 could have different constants. So I don't want that you could create a swerve with swerve1 name and swerve2 constants.

Reply:
maybe they will not have same constnats in sim, like not all arms have same constants in sim

Reply:
So make the sim like real? that it has a swerve name constant?

Reply:
the real like sim, get the name with injection